backend/api/models/Message.py

```diff
@@ -9,22 +9,29 @@ class Message(Mixin, db.Model):
 
     __tablename__ = "message"
 
+    id = db.Column(db.String, unique=True, primary_key=True)
     pm_id = db.Column(db.String, db.ForeignKey("portfolio_manager.id"))
     fp_id = db.Column(db.String, db.ForeignKey("field_partner.id"))
-    to_fp = db.Column(db.Boolean)  # true if send to fp; false if send to pm
+    # true if send to fp; false if send to pm
+    to_fp = db.Column(db.Boolean)
 
-    doc_id = db.Column(db.Integer, unique=True, primary_key=True)
-    status = db.Column(db.String, unique=True)
-    comment = db.Column(db.String, nullable=True)
+    # These are all nullable depending on the type of notification
+    doc_id = db.Column(db.Integer, db.ForeignKey("documents.id"), nullable=True)
+    status = db.Column(db.String, nullable=True)
```

Comment:
what exactly is the status of a message?

Reply:
it's the document status bc all of the current notifs are from doc statuses changing (ik it can just be checked from the doc_id though)

Reply:
is it necessary in that case?

Reply:
no D: i'll try getting rid of using it in the table/passing it into the function in apiwrapper but there miiight be some issues with creating the notif before the document's status is changed